### PR TITLE
[AAP-22421] - public ping endpoint for http healthchecks

### PR DESCRIPTION
**Pull request by msmagnanijr**
Please see:  https://issues.redhat.com/browse/AAP-22421 and https://issues.redhat.com/browse/AAPRFE-1202

### CODE REVIEW

**Review by Alex-Izquierdo — COMMENTED** [review]
we need to expose this view in a public path, /ping/, (right now is only available in /_healthz) 
Also I think it is a good opportunity to improve it adding a basic check of the database connection.
```
    try:
        connection.ensure_connection()
    except OperationalError:
        return HttpResponse("Database connection failed", status=500)
```

**Comment by msmagnanijr** [comment]
> we need to expose this view in a public path, /ping/, (right now is only available in /_healthz) Also I think it is a good opportunity to improve it adding a basic check of the database connection.
> 
> ```
>     try:
>         connection.ensure_connection()
>     except OperationalError:
>         return HttpResponse("Database connection failed", status=500)
> ```

okay, got it!

File: src/aap_eda/core/views.py [hunk 1]
```diff
@@ -1,10 +1,31 @@
...
 
-    def get(self, _request):
-        return Response("OK")
+    def get(self, request):
+        try:
+            connection.ensure_connection()
+            return Response({"status": "OK"}, status=200)
+        except OperationalError:
+            return Response(
+                {"error": "Database connection failed"}, status=500
```
Comment (Alex-Izquierdo) [inline src/aap_eda/core/views.py:17]: Should we include here also the "status" key for consistency?

Comment (msmagnanijr) [reply src/aap_eda/core/views.py]: yep, makes sense!

**Review by ttuffin — APPROVED** [review]

**Pull request merged**